### PR TITLE
old copy and main splitter into base and realsense

### PR DESCRIPTION
**Pull request by iharsuvorau**
Please, check it out. Now there are two main urdf files for robotont description:

- one without the realsense
- the other with the realsense

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotont_description)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
```
Comment (RValner) [inline CMakeLists.txt]: why is the description package depending on any of those packages?

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    joy
+    nav_msgs
+)
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_executable(gazebo_robotont_teleop_joy src/gazebo_robotont_teleop_joy.cpp)
```
Comment (RValner) [inline CMakeLists.txt]: there should be no executables in the description package

**Review by RValner — CHANGES_REQUESTED** [review]
It seems that this commit has many legacy junk that should not be in the "robotont_description" package. Refer to the comments.

**Comment by iharsuvorau** [comment]
Hey, check the new commit 60b28c5.

**Pull request merged**